### DISCUSSION TITLE
PUPPETEER_EXECUTABLE_PATH

### DISCUSSION THREAD

**Discussion by totovic** [jef/streetmerchant#1582] [Help]
Hello, in file .env (macbook pro) i set PUPPETEER_EXECUTABLE_PATH="./Users/salvatore/Chrome.app"
but streetmerchant don't start anyway, the error is:
error :: ✖ something bad happened, resetting streetmerchant in 5 seconds Tried to use PUPPETEER_EXECUTABLE_PATH env variable to launch browser but did not find any executable at: ./Users/salvatore/Chrome.app {
  "stack": "Error: Tried to use PUPPETEER_EXECUTABLE_PATH env variable to launch browser but did not find any executable at: ./Users/salvatore/Chrome.app\n    at ChromeLauncher.launch (/Users/salvatore/streetmerchant/node_modules/puppeteer/lib/cjs/puppeteer/node/Launcher.js:79:23)\n    at async PuppeteerExtra.launch (/Users/salvatore/streetmerchant/node_modules/puppeteer-extra/dist/index.cjs.js:129:25)\n    at async main (/Users/salvatore/streetmerchant/build/index.js:44:15)\n    at async Timeout.loopMain [as _onTimeout] (/Users/salvatore/streetmerchant/build/index.js:81:9)"

why??
where is the problem, or the error?

Thanks 

Regards

**Comment by jef** [comment]
`PUPPETEER_EXECUTABLE_PATH="/Users/salvatore/Chrome.app"`

I believe you don't want the `./` in the beginning.
